samples/core/parameterized_tfx_oss/README.md

```diff
@@ -25,6 +25,9 @@ The notebook sample includes the installation of various dependencies as its fir
 it depends on the latest released KFP and a nightly built TFX to leverage `TFX::RuntimeParameter`.
 
 # Permission
+
+> :warning: If you are using **full-scope** cluster in hosted pipeline beta version, **DO NOT** follow this section.
```

Comment:
**full-scope** or **Workload Identity** cluster?
For WI case, it's the same.

Reply:
Done